### ISSUE TITLE
Make polyglossia options list-aware in LaTeX template

### ISSUE THREAD

**Issue by frederik-elwert** [jgm/pandoc#6443]
In the latex template, most package options are list-aware, i.e., when a package supports multiple options, they can be expressed as a list in the YAML metadata. However, for some reasons, this is not true for the polyglossia(-lang|-otherlangs) options. Thus, instead of writing

```yaml
lang: de
polyglossia-lang:
    name: german
    options:
        - spelling=new
        - babelshorthands=true
```

one currently has to write

```yaml
lang: de
polyglossia-lang:
    name: german
    options: spelling=new,babelshorthands=true
```

Since this makes the interface for LaTeX package options incoherent, I think this should be aligned.

**Comment by cderv** [comment]
FYI Just passing by here looking for some information, and it seems this issue has been fixed by #6444 so probably can be closed.

**Comment by tarleb** [comment]
Thanks! What's more is that pandoc now uses babel instead of polyglossia for all LaTeX engines.
